File: MORM.Java/src/MORM/Java/Models/Venda/Aliqicms.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "ALIQICMS")
public class Aliqicms extends CollectionItem
{
    @CampoAttribute(Campo = "UF_ORIGEM", Tipo = CampoTipo.Key)
    public String Uf_Origem;
    @CampoAttribute(Campo = "UF_DESTINO", Tipo = CampoTipo.Key)
    public String Uf_Destino;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "PR_ALIQUOTA", Tipo = CampoTipo.Req)
    public Float Pr_Aliquota;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Aliqicms.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("ALIQICMS")]
    public class Aliqicms : CollectionItem
    {
        [Campo("UF_ORIGEM", CampoTipo.Key)]
        public string Uf_Origem { get; set; }
        [Campo("UF_DESTINO", CampoTipo.Key)]
        public string Uf_Destino { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("PR_ALIQUOTA", CampoTipo.Req)]
        public double Pr_Aliquota { get; set; }
    }
}